### PR TITLE
Put notes where they render correctly

### PR DESCRIPTION
**Pull request by MSeifert04**
Unfortunately I still can't build the docs but currently they don't render correctly. Could someone check if they render correctly with the PR locally?

### CODE REVIEW

**Comment by bsipocz** [comment]
@MSeifert04 - What's the issue with building the docs? are you around on slack to chat about it?

**Comment by MSeifert04** [comment]
I already reported it in #6677. I'll probably have time later today but just now I have some appointments. :(

**Comment by pllim** [comment]
So does this fix #6677? If so, I can take this for a spin on my Windows laptop if you need me to.

**Comment by MSeifert04** [comment]
No it doesn't fix #6677. It just fixes some doc-render-problems: http://docs.astropy.org/en/stable/api/astropy.table.Table.html#astropy.table.Table and http://docs.astropy.org/en/stable/api/astropy.table.Table.html#astropy.table.Table.copy

File: astropy/table/table.py [hunk 1]
```diff
@@ -207,16 +207,17 @@ class Table:
         Metadata associated with the table.
     copy : bool, optional
         Copy the input data. Default is True.
+
```
Comment (mhvk) [inline astropy/table/table.py]: If things do not work, I think it is completely fine to just add this note directly to the text, without `.. note`

**Comment by MSeifert04** [comment]
@mhvk You're right. No need to spill notes all around the place. It's also more readable 👍

**Pull request merged**